### PR TITLE
Update structure-of-a-contract.rst

### PR DESCRIPTION
**Pull request by michaltrzesimiech**
A minor fix to 'Function Modifiers'

### CODE REVIEW

File: docs/structure-of-a-contract.rst [hunk 1]
```diff
@@ -71,7 +71,7 @@ Function modifiers can be used to amend the semantics of functions in a declarat
     }
   }
 
- in the section on contracts for a more in-depth explanation.
+See into the section on :ref:`contracts` for a more in-depth explanation.
```
Comment (chriseth) [inline docs/structure-of-a-contract.rst]: Isn't that redundant to what is said before the example?

Comment (VoR0220) [inline docs/structure-of-a-contract.rst]: except that it makes a reference...which is nice to have.

Comment (chriseth) [inline docs/structure-of-a-contract.rst]: It also has a reference. So, @michaltrzesimiech thanks for the PR, could you please change it to just delete that line?

**Comment by chriseth** [comment]
Perfect, thanks a lot!

**Pull request merged**